webapp/src/ts/modules/messages/messages.component.ts

```diff
@@ -83,12 +75,14 @@ export class MessagesComponent implements OnInit, OnDestroy {
       loadingContent,
       error,
     ]) => {
+      this.selectedConversationId = selectedConversation?.id;
```

Comment:
I don't think `selectedConversationId` needs to be an instance variable. Is doesn't appear to be used anywhere.

Reply:
Oh yes, that can be removed. Done!